src/app/containers/SettingsPage/index.js

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
should use `import * as React` to get all the type recommendations. For `useEffect` lets just type it out a full path like `React.useEffect`

```
https://stackoverflow.com/questions/54585763/what-is-the-difference-between-import-as-react-from-react-vs-import-react-fr/54585795
Pattern import * as React from 'react is related to the use of type systems in React like Flow or Typescript. Using import React from 'react' has led to issues with importing types definitions
```